### DISCUSSION TITLE
Is the old port of nrf52840 now redundant?

### DISCUSSION THREAD

**Discussion by zhitaoh** [contiki-ng/contiki-ng#1986] [Q&A]
It appears that the nrf52840 platform has two alternative, but largely similar ports. Does the newer one at _arch/{cpu|platform}/nrf/nrf52840_ supercede the older one at _arch/{cpu|platform}/nrf52840_?

**Comment by alexstanoev** [comment]
Eventual feature parity is the goal for the two nRF52840 ports, however, in their current state:
- You would pick the `nrf52840` port if you require native USB CDC-ACM support, or require a module provided by the nRF5 SDK, in exchange for no nRF53 support.
- You would pick the `nrf` port if your application does not make use of USB, and does not need any libraries provided by the nRF5 SDK - any peripheral interactions would be covered by the `nrfx` HAL. You will get code portability between the nRF52840 and nRF5340.

**Reply by pjonsson** [reply]
Does #1609 make the USB-support of the `nrf` port on par with the `nrf52840` port?

**Reply by Yagoor** [reply]
Not yet. It is still missing the DFU support but I want to add that on #1609
So the only difference will be that it does not contain the functionalities from the nRF5 SDK.
